open_schools_platform/common/admin.py

```diff
@@ -0,0 +1,17 @@
+from django.contrib import admin
+
+
+class InputFilter(admin.SimpleListFilter):
+    template = 'templates/input_filter.html'
+
+    def lookups(self, request, model_admin):
+        return ((),)
+
+    def choices(self, changelist):
+        all_choice = next(super().choices(changelist))
+        all_choice['query_parts'] = (
+            (k, v)
+            for k, v in changelist.get_filters_params().items()
+            if k != self.parameter_name
+        )
+        yield all_choice
```

Comment:
Do you understand how yield works? It is not mistake, just question.

Reply:
it works similar to return method, but yield returns values, that are not saved anywhere (generated values).